code/modules/mob/living/carbon/xenomorph/abilities/queen/queen_abilities.dm

```diff
@@ -63,6 +63,11 @@
 	action_icon_state = "xeno_readmit"
 	plasma_cost = 100
 
+
+/datum/action/xeno_action/onclick/manage_hive
+	name = "Manage The Hive"
+	action_icon_state = "xeno_readmit"
```

Comment:
A sprite for this would be nice.

Reply:
For the icon state? i mean, the other ones are no longer used so i just used one of the ones that arent use danymore